docs/usage_guide.rst

```diff
@@ -9,29 +9,53 @@ Introduction
 
 In this section we are going to see how to use various parts of the Rotki application.
 
-Sign-in/Signup
-==================
+
+First time sign-up
+====================
 
 When you start Rotki you are greeted with a sign-in/signup prompt.
 
 .. image:: images/sc_newacc1.png
    :alt: Creating a new account
    :align: center
 
-If you have never created an account before press "Create New Account". Provide a username and a password. Don't forget this password. It will be used to encrypt all your local files. The username is just an identifier for your database. It's a local user.
+.. role:: red
+
+For creating an account press "Create New Account" and provide a username and a password:
+
+  - **Username**: it is just an identifier for your database; a local user.
+  - **Password**: :red:`Do not forget this password`. It is used to encrypt all your local files. For premium subscription, it also allows to save your database in the Rotki Server and sync it in multiple devices.
+
+If you have purchased a premium subscription you can also add the **API Key** and the **secret** here. See the section `Sync data with Rotki Server <sync_data_with_rotki_server_>`_ to know how to sync your data with Roki Server (this option is disabled by default).
 
 .. image:: images/sc_newacc2.png
    :alt: Creating a new account with a premium Rotki API key/secret pair
    :align: center
 
-If you have purchased a premium subscription you can also add the API Key and the secret here. This will essentially populate the new account with the data saved in the server for your premium subscription. Important thing to note here is that the password of the new account must be the same as the data saved in the server. If not, opening the database will fail.
+All accounts are created in the rotki directory which is located in: ``$HOME/.rotkehlchen``. Each user has their own subdirectory.
```

Comment:
Hmm this is wrong and outdated. Can you change it? The location of the rotki directory depends on the OS. The locations are in the documentation here: https://rotki.readthedocs.io/en/latest/usage_guide.html?highlight=Library#rotki-data-directory